storageClient/storageClient.go

```diff
@@ -58,6 +58,20 @@ func (c *StorageClient) CreateGroup(ctx context.Context, params CreateGroupReque
 	return result, err
 }
 
+// DescribeGroup fetches a group using the specified parameters,
+// returning the group and error (if any).
+func (c *StorageClient) DescribeGroup(ctx context.Context, params DescribeGroupRequest) (*Group, error) {
+	var result *Group
+	path := c.Host + storageServiceBasePath + "/groups"
```

Comment:
+ "/" + params.GroupID

Reply:
it would be params.Groupid.String() right?